source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -262,7 +262,7 @@ namespace
         for (RtPrim masterPrim : RtApi::get().getMasterPrims(nodedefFilter))
         {
             RtNodeDef candidate(masterPrim);
-            if (candidate.getNode() == nodeName &&
+            if (candidate.getNamespacedNode() == nodeName && 
```

Comment:
Fix to compare name masterprim name against namespaced name as materprim store namespaced names now.